dawn/renderer/components/Editor.js

```diff
@@ -128,7 +131,7 @@ class Editor extends React.Component {
   }
 
   getEditorHeight(windowHeight) {
-    const windowNonEditorHeight = 231 + (this.props.showConsole * (this.consoleHeight + 40));
+    const windowNonEditorHeight = 235 + (this.props.showConsole * (this.state.consoleHeight + 40));
```

Comment:
Is there a way to not have these hardcoded constants here? 
Some global variable? 

Not sure if this.state is similar.